generate.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2021 Pyrra Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+)
+
+func cmdGenerate(logger log.Logger, configFiles, prometheusFolder string) int {
+	fs, err := ioutil.ReadDir(configFiles)
```

Comment:
As done by @Gaikanomer9 in #365 it would be great to use a `filepath.Glob` here:
https://github.com/pyrra-dev/pyrra/pull/365/files#diff-daddc66424f674dc57b411e63fce1f1bd231a27a77e39d25c781b4a8ca6f6062R11